Tests/scripts/create_artifacts_graph/create_artifacts.py

```diff
@@ -23,9 +23,13 @@ def create_dependencies(content_dto: ContentDTO, output: Path):
     for pack in content_dto.packs:
         dependencies = pack.depends_on
         first_level_dependencies = {}
-        all_level_dependencies = []
+        all_level_dependencies = {}
         for dependency in dependencies:
-            all_level_dependencies.append(dependency.content_item.object_id)
+            if dependency.mandatorily:
+                all_level_dependencies[dependency.content_item.object_id] = {
+                    "display_name": dependency.content_item.name,
+                    "mandatory": dependency.mandatorily,
+                }
```

Comment:
```suggestion
            if dependency.mandatorily:
                all_level_dependencies[dependency.content_item.object_id] = {
                    "display_name": dependency.content_item.name,
                    "mandatory": True,
                }
```
If you check the mandatorily so it always will be true. no?